metrics/lib/send_results.sh

```diff
@@ -36,7 +36,7 @@ COR_DEFAULT_PATH="/usr/bin"
 COR_DEFAULT_NAME="cc-oci-runtime"
 RUNC_DEFAULT_PATH="/usr/bin"
 RUNC_DEFAULT_NAME="docker-runc"
-CC_RUNTIME_DEFAULT_PATH="/usr/local/bin"
+CC_RUNTIME_DEFAULT_PATH="/usr/bin"
```

Comment:
So, iirc, the difference between having `local` or not in the path is that between a, shall we say, development install and a production(release) install. As long as this path is working for the CI testing infra, then I'm happy with the change :-)